client/src/containers/EditUserProfile/styles.css

```diff
@@ -55,6 +69,33 @@
     font-size: 16px;
 }
 
+.edit-user-profile-container .user-avatar-controls {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding: 20px;
+}
+
+.edit-user-profile-container .user-avatar-controls .update-user-avatar {
```

Comment:
To avoid duplication of code:
.edit-user-profile-container .user-avatar-controls .button {
    // all common styles
}

.type-update-avatar {
    background-color: #02bcd4;
}

.type-delete-avatar {
    background-color: orange;
}

Reply:
Yes, thank you! It is a good improvement! 👍